### PR TITLE
Correct flux patch config gor PIP and HMI

### PR DESCRIPTION
**Pull request by chris-pateman**
### Change description ###
Correct the flux patch location for pip and hmi

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

File: k8s/environments/sbox/common-overlay/flux-patch.yaml [hunk 1]
```diff
@@ -18,3 +18,97 @@ spec:
...
+    fluxcd.io/automated: "true"
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/pip/channel-management:latest
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: hmi-case-hq-emulator
```
Comment (timja) [inline k8s/environments/sbox/common-overlay/flux-patch.yaml:109]: did you mean to add this here?

File: k8s/environments/stg/common-overlay/flux-patch.yaml [hunk 2]
```diff
@@ -51,13 +51,13 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   annotations:
-    filter.fluxcd.io/nodejs: glob:prod-*
+    filter.fluxcd.io/nodejs: glob:aat-*
```
Comment (timja) [inline k8s/environments/stg/common-overlay/flux-patch.yaml:54]:
```suggestion
    filter.fluxcd.io/nodejs: glob:prod-*
```

File: k8s/environments/stg/common-overlay/flux-patch.yaml [hunk 3]
```diff
@@ -51,13 +51,13 @@ apiVersion: helm.fluxcd.io/v1
...
   name: pip-frontend
   namespace: pip
 spec:
   values:
     nodejs:
-      image: sdshmctspublic.azurecr.io/pip/frontend:prod-51d4bf6-20210826152156
+      image: sdshmctspublic.azurecr.io/pip/frontend:aat-51d4bf6-20210826152156
```
Comment (timja) [inline k8s/environments/stg/common-overlay/flux-patch.yaml:60]:
```suggestion
      image: sdshmctspublic.azurecr.io/pip/frontend:prod-51d4bf6-20210826152156
```

File: k8s/environments/sbox/common-overlay/flux-patch.yaml (hunk 1 above)
Comment (chris-pateman) [reply k8s/environments/sbox/common-overlay/flux-patch.yaml:109]: yes.... I believe. It is another project that needs to go up and was implemented in the same method as the pip-frontend

File: k8s/environments/stg/common-overlay/flux-patch.yaml (hunk 3 above)
Comment (chris-pateman) [reply k8s/environments/stg/common-overlay/flux-patch.yaml:60]: do I not want to use latest here?

Comment (timja) [reply k8s/environments/stg/common-overlay/flux-patch.yaml:60]: no latest is just for initial use so you don't have to bother looking up the image tag, flux will write the actual latest version here, `latest` isn't recommended for actual running

**Review by timja — APPROVED** [review]

**Pull request merged**